docs/src/pages/ColorsPage/ColorsPage.js

```diff
@@ -84,8 +84,8 @@ const ColorsPage = () => (
     }}
     sections={sections}
     nextPageLink={{
-      title: 'Typography',
-      link: ROUTES.TYPESETTING,
+      title: 'Copywriting',
+      link: ROUTES.COPYWRITING,
```

Comment:
This is not part of the new docs site, can you revert please?

Reply:
So it's meant to go to https://backpack.github.io/tokens/typesetting/